app/models/user.rb

```diff
@@ -77,4 +77,8 @@ def request_admin_status(organisation_id)
   def self.purge_deleted_users_where(query)
     User.deleted.delete_all(query)
   end
+
+  def superadmin_emails
```

Comment:
This method returns the email addresses of superadmin users that's why it can't be associated to a user instance.